impeller/typographer/backends/skia/typographer_context_skia.cc

```diff
@@ -123,16 +196,12 @@ static ISize OptimumAtlasSizeForFontGlyphPairs(
     const std::vector<FontGlyphPair>& pairs,
     std::vector<Rect>& glyph_positions,
     const std::shared_ptr<GlyphAtlasContext>& atlas_context,
-    GlyphAtlas::Type type,
     const ISize& max_texture_size) {
-  static constexpr auto kMinAtlasSize = 8u;
-  static constexpr auto kMinAlphaBitmapSize = 1024u;
+  static constexpr auto kMinAtlasSize = 4096u;
```

Comment:
Because we don't have to upload the whole bitmap for each change, we can afford a larger texture.

Reply:
backed this out for now because I'd need to rework all of the tests.